lambdas/preview/index.py

```diff
@@ -23,6 +24,8 @@
 MAX_LINES = 512 # must be positive int
 MIN_VCF_COLS = 8 # per 4.2 spec on header and data lines
 
+S3_DOMAIN_SUFFIX = '.s3.amazonaws.com'
```

Comment:
are we sure we can't get path-style requests?

Reply:
Pretty sure... I was actually trying to get JS to make path-style requests cause it would make the S3 proxy simpler, but couldn't find a way.